src/sentry/discover/compare_timeseries.py

```diff
@@ -103,9 +106,11 @@ def make_rpc_request(
         config=SearchResolverConfig(
             extrapolation_mode=extrapolation_mode,
         ),
-        sampling_mode=None,
+        sampling_mode="NORMAL",
     )
 
+    sentry_sdk.set_tag("extrapolation_mode", extrapolation_mode_str)
```

Comment:
I call this `query.sampling_mode` [here](https://github.com/getsentry/sentry/blob/master/src/sentry/snuba/rpc_dataset_common.py#L216)